database/factories/DraftFactory.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Database\Factories;
+
+use App\Models\User;
+use App\Models\Team;
+use Illuminate\Support\Str;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+/**
+ * @extends \Illuminate\Database\Eloquent\Factories\Factory<\App\Models\Draft>
+ */
+class DraftFactory extends Factory
+{
+    /**
+     * Define the model's default state.
+     *
+     * @return array<string, mixed>
+     */
+    public function definition()
+    {
+        $user = User::factory()->withPersonalTeam()->create();
+        $key = Str::uuid();
+        $environment = env('APP_ENV', 'local');
+        $path = $environment.'/'.$user->id.'/drafts/'.$key;
+
+        return [
+            'name'            => Str::random(),
+            'slug'            => Str::random(),
```

Comment:
I have defined the name and slug for the project like this:
![Uploading image.png…]()